### PR TITLE
feat(components): Add CollapsibleCard

### LINKED ISSUE #62: Collapsible Cards
This is needed for the following patterns:

1. `DataGrid[type=AdvancedSearch]` to wrap the form-based search params
1. `Rolodex` a container for `Card`s (e.g.; CANs impl)

### PR DESCRIPTION
**Pull request by eckdanny-osi**
Closes #62 

collapsiblecards

### CODE REVIEW

**Comment by eckdanny-osi** [comment]
^ @kelfink

File: libs/components/src/CollapsibleCard/CollapsibleCard.module.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.HeaderWrapper {
+  display: flex;
+  flex-wrap: nowrap;
+}
+
+.HeaderChildrenWrapper {
+  flex: 1;
+}
```
Comment (NagaPradeepGajula-osi) [inline libs/components/src/CollapsibleCard/CollapsibleCard.module.scss]: did we verify these flex styles in IE and edge?

File: libs/components/src/CollapsibleCard/CollapsibleCard.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cn from 'classnames'
+import { Button, Card, Collapse, CardHeader } from '@cwds/reactstrap'
+import Icon from '@cwds/icons'
+// import IconButton from '../IconButton'
```
Comment (NagaPradeepGajula-osi) [inline libs/components/src/CollapsibleCard/CollapsibleCard.jsx]: NIT: commented code can be removed.

File: libs/components/src/CollapsibleCard/CollapsibleCard.module.scss (hunk 1 above)
Comment (eckdanny-osi) [reply libs/components/src/CollapsibleCard/CollapsibleCard.module.scss]: flexbox layouts is core to BS4. [`webpacker` postcss config](https://github.com/rails/webpacker/blob/8d496a0b871085648d6ee21c0e1e469d11cfe593/lib/install/config/postcss.config.js#L4-L10) and [`CRA@2` postcss config](https://github.com/facebook/create-react-app/blob/34e5029c548f0fa08e1cd0a9865fc7c90914ff1e/packages/react-scripts/config/webpack.config.js#L103-L109) both include by default `postcss-flexbugs-fixes` and `postcss-preset-env`, so the `browserlist` should automagically include vendor prefixes or sugar the declarations at app compile time.

File: libs/components/src/CollapsibleCard/CollapsibleCard.jsx (hunk 2 above)
Comment (eckdanny-osi) [reply libs/components/src/CollapsibleCard/CollapsibleCard.jsx]: shall remove...

**Pull request merged**